src/notification/types.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import type {Node} from 'react';
+import {KIND} from './constants';
+import type {ThemeT} from '../styles/types';
+import type {OverrideT} from '../helpers/overrides';
+
+export type OverridesT = {
+  Root?: OverrideT<*>,
+};
+
+export type NotifcationPropsT = {
+  children?: Node,
+  overrides?: OverridesT,
+  kind: $Keys<typeof KIND>,
```

Comment:
$Values is more correct, but we need to freeze it https://github.com/uber-web/baseui/issues/539

Reply:
We need to make this update in several areas. The majority of components are using `$Keys`.

Reply:
Updated and frozen.